pyigm/surveys/lls_literature.py

```diff
@@ -0,0 +1,1078 @@
+"""
```

Comment:
Is this the code you used to send me the tables for the LLS paper? I recall I had to clean a couple of things (can't remember what, though). If so, worth double checking the outputs to make sure there are no double systems or spurious ions.